pkg/data/components/OTelGRPCExporter.yaml

```diff
@@ -59,6 +59,36 @@ properties:
     type: bool
     default: false
     advanced: true
+  - name: BatchTimeout
+    summary: How long to wait to before sending a batch, regardless of size.
+    description: |
+      Configure how long to wait before sending a batch. The batch will be sent after
+      this timeout.
+    type: duration
+    default: 60s
+    validations:
+      - duration
+      - nonempty
+    advanced: true
+  - name: BatchSize
+    summary: The size of a batch.
+    description: |
+      The size of a batch, measured by span/datapoint/log record count. Once a batch reaches this size it will be sent.
+    type: int
+    default: 100_000
+    validations:
+      - nonempty
+    advanced: true
+  - name: QueueSize
+    summary: The size of a exporting queue.
+    description: |
+      The size of the exporting queue, measured by span/datapoint/log record count.
+      Items will be kept in the queue while the batch is being created.
+    type: int
+    default: 1_000_000
+    validations:
+      - nonempty
+    advanced: true
```

Comment:
These default made sense for the awss3exporter for data activation, but are likely too large for other things. Lets set the defaults to match the batch processor defaults:

- max size: 8192
- flush timeout: 200ms
- queue size: probably something much larger than 8192. maybe 100_000?

Reply:
updated defaults